### DISCUSSION TITLE
🔥 🤯 🚀 GitHub as a blogging  service

### DISCUSSION THREAD

**Discussion by dillingham** [community/community#8361] [Other Feature Feedback, Questions, & Ideas]
if you added [yaml metadata](https://yaml.org/spec/1.2-old/spec.html#id2760395) support or meta tag fields as part of the commit dialog for `.md` files...

and added a repository setting for "Show repo as blog" which hides many tabs and displays as an index vs file directory...

Github would instantly be a blogging platform...that allows contributions from the audience...

- You already have blog subscriptions via "Watch" (just make RSS of what they watch)
- You already have comments via "Discussions"
- You already have analytics via "Insights"

**Comment by dillingham** [comment]
Slightly related: instead of wiki as is.. allow adding a github repo as another repo's wiki in setttings

lets others PR fixes and just becomes a content display of a repo (like  this task)

**Comment by serpro69** [comment]
I was also thinking about using github as a blogging platform, but using a static site generator like Jekyll or similar. I would be interested to hear how an implementation of this feature request would be better than a static site generated from markdown files (which are still open and can have contributions from the audience so long as your repo is public and allows contributions).

**Reply by dillingham** [reply]
Good points, I would say the "create repo and be done" flow would be a big  sell for me.
- No build steps
- No hosting
- Discoverability provided github syndicates all the content into a feed / directory 

Last bullet point: imagine https://hashnode.com/explore

Then could explore other Github established features like
- "Watch" or "Follow" author for email notifications
- Sponsor only content 
- One time sponsor tips
- Analytics 
- Discussions
- Linked issues / PRs / releases in sidebar of article
- Linked articles in sidebar of issue / PRs  / releases

Last points are great for project promotion or post mortem

**Reply by serpro69** [reply]
Yeah, I can see how "no build, no hosting, no fuss, just write" can be a selling point for attracting non-technical writers. Not sure it's enough to "sell" this feature request to github though :) but I think would be nice if they did consider it

**Reply by dillingham** [reply]
That's would be the sell for the developers or dev relation staff writing technical things for their team or open source. The sell for GitHub would be the same as why they built discussions when we have stack overflow.. take over the dev space and get access to private data so they can take over the world 😈
